changelog.d/898.change.rst

```diff
@@ -1 +1,2 @@
 Speedup instantiation of frozen slotted classes.
+If an eq key is defined, it is also used before hashing the attribute.
```

Comment:
this is not how our changelog works 😅

please create a new file `changelog.d/909.change.rst` with the contents.

see:  https://github.com/python-attrs/attrs/blob/main/.github/CONTRIBUTING.md#changelog